File: Sakila_Net/Sakila_Net/VB_SakilaLib/DAO/DAOCustomer.vb
'using System;
Imports  System.Data
'using System.Collections.Generic;
'using System.Configuration;
'using System.Collections;
Imports  SakilaLib.Cl.Harrys.Sakila.Model
'using Cl.Harrys.BD;

Namespace Cl
    Namespace Harrys
        Namespace Sakila
            Namespace DAO

                Public Class DAOCustomer
                    'Inherits SakilaLib.Cl.Harrys.Sakila.DAOBD.DAOCustomer
                    Public Shared  Function SelectAll(nPagina as Integer ,cantidad as Integer) as DataTable
                        Return DAOBD.DAOCustomer.SelectAll(nPagina, cantidad)	   
                    End Function
                    Public Shared  Function SelectOne( customer as ClCustomer) as Boolean
                        Return DAOBD.DAOCustomer.SelectOne(customer)     
                    End Function
                    Public Shared  Function SqlInsert( customer as ClCustomer) as Integer
                        Return DAOBD.DAOCustomer.SqlInsert(customer)     
                    End Function
                    Public Shared  Function SqlDelete( customer as ClCustomer) as Integer
                        Return DAOBD.DAOCustomer.SqlDelete(customer)     
                    End Function
                    Public Shared  Function SqlUpdate( customer as ClCustomer) as Integer
                        Return DAOBD.DAOCustomer.SqlUpdate(customer)     
                    End Function
                End Class

            End Namespace  
        End Namespace  
    End Namespace  
End Namespace  

File: Sakila_Net/CS_SakilaLib/DAO/DAOCustomer.cs
//using System;
using System.Data;
//using System.Collections.Generic;
//using System.Configuration;
//using System.Collections;
using Cl.Harrys.Sakila.Model;
namespace  Cl.Harrys.Sakila.DAO
{   
  public class DAOCustomer 
  {
       public static DataTable sqlSelectAll(int nPagina,int cantidad)
	   {
           return DAOBD.DAOCustomer.selectAll(nPagina,cantidad);
	   }
       public static bool sqlSelect(ClCustomer customer)
	   {
            return DAOBD.DAOCustomer.SelectOne(customer); 
	   }
	   public static int sqlInsert(ClCustomer customer)
	   {
            return DAOBD.DAOCustomer.sqlInsert(customer);
       }
	   public static int sqlDelete(ClCustomer customer)
	   {
            return DAOBD.DAOCustomer.sqlDelete(customer);
       }
	   public static int sqlUpdate(ClCustomer customer)
	   {
            return DAOBD.DAOCustomer.sqlUpdate(customer);
       }
   }
}
